src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,6 +14,8 @@
     private Phone phone;
     private Email email;
     private Address address;
+    public int sequenceNumber;
+    public static int nextSequenceNumber = 0;
```

Comment:
don't you think it make more sense semantically to have `nextSequenceNumber` start from 1